sunpy/map/mapbase.py

```diff
@@ -531,6 +531,7 @@ def dtype(self):
         return self.data.dtype
 
     @property
+    @deprecated(since="2.1", message="Use map.data.shape instead", alternative="map.data.shape")
```

Comment:
```suggestion
    @deprecated(since="2.1", message="Use map.data.size instead", alternative="map.data.size")
```

Reply:
I guess the issue was to use map.data.shape instead of map.size property? or is it map.data.size?